scss/sections/guide.scss

```diff
@@ -0,0 +1,7 @@
+.steps {
+  padding: 50px;
+}
+
+.step {
```

Comment:
sass enables you to nest classes inside of eachother. that way it will be like `.steps .step {...}` by coding


```css
steps {
   .step {}
}
```